api/db/db_utils.py

```diff
@@ -49,7 +51,10 @@ def bulk_insert_into_db(model, data_source, replace_on_conflict=False):
         with DB.atomic():
             query = model.insert_many(data_source[i:i + batch_size])
             if replace_on_conflict:
-                query = query.on_conflict(preserve=preserve)
+                if isinstance(DB, PooledMySQLDatabase):
```

Comment:
@Feiue I'm fix the issue about:

```
ValueError('Postgres requires that a conflict target be specified when doing an upsert.')
```

If we used POSTGRES, must set the  `conflict target ` (version > 9.5), but if we used MYSQL,  the `conflict target ` can not be set, otherwise will meet this issue:

```
MySQL does not support the specification of where clauses or conflict targets for conflict resolution.
```

We have to set the different configurations for DB.